nailgun/entities.py

```diff
@@ -2864,8 +2864,13 @@ def path(self, which=None):
     def read(self, entity=None, attrs=None, ignore=None):
         """Fetch an attribute missing from the server's response.
 
+        Also add sync plan to the responce if needed, as
+        :meth:`nailgun.entity_mixins.EntityReadMixin.read` can't initialize
+        sync plan.
+
         For more information, see `Bugzilla #1237283
-        <https://bugzilla.redhat.com/show_bug.cgi?id=1237283>`_.
+        <https://bugzilla.redhat.com/show_bug.cgi?id=1237283>`_ and
+        `nailgun#261 <https://github.com/SatelliteQE/nailgun/issues/261>`_.
```

Comment:
My idea was to include some additional information of what's done in this method and why. PR has a link to such info, but `Product.read()` had no connection with either PR or issue, so i've added it and i think it may be useful. Most of nailgun methods with some workarounds have a link to detailed description so it's a common approach here, although your point is good, we don't usually link nailgun issues. Anyway, 1:1 here, let's allow the second reviewer to decide :smile: